src/struct/dwarves.c

```diff
@@ -2222,11 +2222,11 @@ void cus__delete(struct cus *cus)
 	free(cus);
 }
 
-void dwarves__fprintf_init(uint16_t user_cacheline_size);
```

Comment:
Again, it seems maybe bad to be making refactoring changes to what is mostly a drop-in file from pahole.  Not a huge deal, but this kind of stuff accumulates eventually.

Reply:
I've reverted these changes in the latest version.  It should be much less invasive now.